pkg/plugin/plugin.go

```diff
@@ -121,14 +121,17 @@ func makeAutoscaleEnforcerPlugin(ctx context.Context, obj runtime.Object, h fram
 	submitVMBoundsChanged := func(vm *api.VmInfo, podName string) {
 		pushToQueue(func() { p.handleUpdatedScalingBounds(vm, podName) })
 	}
+	submitNonAutoscalingVmUsageChanged := func(newVM *api.VmInfo, name util.NamespacedName) {
```

Comment:
Another comment on this: The pod's namespace is guaranteed to be the same as the VM's (i.e. `newVM.Namespace == name.Namespace` always holds). For that reason, the other handler functions just take a `string` instead of a `util.NamespacedName`

(also `newVM` looks left over from earlier commits?)